Cabal/doc/developing-packages.rst

```diff
@@ -214,9 +262,14 @@ the package:
 
 .. code-block:: console
 
-    $ cabal configure
     $ cabal build
 
+As mentioned above, if you have an executable you can also run it with:
+
+.. code-block:: console
+
+   $ cabal run
+
 Assuming those two steps worked then you can also install the package:
 
 .. code-block:: console
```

Comment:
Does this work? It would be nice if it worked, but i'm 100% that installing local llbraries doesn't. (There should be a note, that it will work eventually, but doesn't now).

Reply:
Hmm, I don't think so, but I'll need to investigate further. Going to call this out of scope of this change, can follow-up afterwards.

Reply:
Add

```
.. todo:: this might or might not work with v2-install
```

I don't believe in follow-ups. Better to leave a sign.

Reply:
Added todo.